### PR TITLE
go_1_13: 1.13.6 -> 1.13.7, go_1_12: 1.12.15 -> 1.12.16

### PR DESCRIPTION
**Pull request by Frostman**
###### Motivation for this change

Go 1.13.7 and 1.12.16 (released 2020/01/28) includes two security fixes to the crypto/x509 package

Go itself works fine, but it seems like there are a few packages failing unrelated to the change, currently investigating.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/78991)

### CODE REVIEW

**Comment by kalbasit** [comment]
@Frostman what are the failing packages?

**Comment by Frostman** [comment]
@kalbasit `aria2p` and `editorconfig-checker` are failing:

```
hash mismatch in fixed-output derivation '/nix/store/6ghdnnzr7w8ah9p6pym8mik67bp580jy-editorconfig-checker-2.0.2-go-modules':
  wanted: sha256:09b1v9gyh6827yqlfxxxq3lcqhd5snn3n7gdlbjmga3wyp2x4g2r
  got:    sha256:1iiv12ginb3ky739z7v8wf4z5lv24gmghbybs3lzay0kqn449n4x
cannot build derivation '/nix/store/dhickjqkym3hgr7ispji7yi7argwbnpj-editorconfig-checker-2.0.2.drv': 1 dependencies couldn't be built
builder for '/nix/store/k2x86lq0csx8gvbxd9yqzfl9x60k55d8-python3.8-aria2p-0.7.0.drv' failed with exit code 2; last 10 log lines:
    File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
    File "<frozen importlib._bootstrap>", line 991, in _find_and_load
    File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
    File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
    File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
    File "<frozen importlib._bootstrap>", line 991, in _find_and_load
    File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
  ModuleNotFoundError: No module named 'poetry'
```

and `python3.7-python-hglib-2.6.1` is stuck on running install tests if I'm reading it correctly:

```
[1/47/50 built (2 failed), 79 copied (511.0/511.4 MiB), 202.0 MiB DL] building python3.7-python-hglib-2.6.1 (installCheckPhase): running install tests
```

(not changing for 30+ minutes)

**Comment by Frostman** [comment]
I did another rebuild with the current master and there are the same issues.

* here is the bump for `editorconfig-checker` - https://github.com/NixOS/nixpkgs/pull/79145
* I'm pretty sure that python packages rebuild couldn't be broken by go bump, so, I believe this patch should be safe

@kalbasit I'm very new for NixOS, so, would appreciate it if you can share some ideas on the next steps or even if it's a blocker or not. thanks!

File: pkgs/development/compilers/go/1.13.nix [hunk 1]
```diff
@@ -30,11 +30,11 @@ in
 
 stdenv.mkDerivation rec {
   pname = "go";
-  version = "1.13.6";
+  version = "1.13.7";
 
   src = fetchurl {
     url = "https://dl.google.com/go/go${version}.src.tar.gz";
-    sha256 = "03220q6n4wlpmz6zz3mw48kl3pjxia6pxdvf03wbqh6w9favxrda";
+    sha256 = "1x21kfpzfkvmqd42pan6nl862m7jjl4niqxxpcgm46awbz645bg4";
```
Comment (kalbasit) [inline pkgs/development/compilers/go/1.13.nix]: This commit should be `go_1_13: 1.13.6 -> 1.13.7`, please rebase and force-push.

**Review by kalbasit — CHANGES_REQUESTED** [review]

**Comment by kalbasit** [comment]
@GrahamcOfBorg build go_1_12 go_1_13 pet jx

**Review by kalbasit — APPROVED** [review]

**Pull request merged**